### PR TITLE
Handling dummy symbols in Length proc

### PR DESCRIPTION
**Pull request by vhavlena**

### CODE REVIEW

File: src/smt/theory_str_noodler/aut_assignment.cpp [hunk 1]
```diff
@@ -208,7 +208,7 @@ namespace smt::noodler {
     }
 
     void AutAssignment::replace_dummy_with_new_symbol() {
-        mata::Symbol new_symbol = regex::Alphabet(alphabet).get_unused_symbol();
+        mata::Symbol new_symbol = alphabet.size() == 0 ? 92 : regex::Alphabet(alphabet).get_unused_symbol();
```
Comment (jurajsic) [inline src/smt/theory_str_noodler/aut_assignment.cpp:211]: If the alphabet size is 0, then there cannot be a dummy symbol, I would just return.

Comment (vhavlena) [reply src/smt/theory_str_noodler/aut_assignment.cpp]: Yes, right

**Review by jurajsic — APPROVED** [review]
Looks good, will you try running it? I assume this should break anything.

**Comment by vhavlena** [comment]
> Looks good, will you try running it? I assume this should break anything.

I am about to skip it. It should break nothing (hopefully).

**Comment by jurajsic** [comment]
> Looks good, will you try running it? I assume this should break anything.

*shouldn't :D

Ok, you can merge.

**Pull request merged**